### PR TITLE
New Tweak: Limit Target Status Effects

### PR DESCRIPTION
**Pull request by Aireil**
👋 
If you liked the idea, here it is.

### CODE REVIEW

**Comment by Caraxi** [comment]
It doesn't appear to be affecting the list when the target info display is in merged mode?

**Comment by Aireil** [comment]
Good catch, fixed 👍

File: Tweaks/UiAdjustment/LimitTargetStatusEffects.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        }
+
+        public override void Disable() {
+            PluginInterface.Framework.OnUpdateEvent -= FrameworkOnUpdate;
+            UpdateTargetStatus(true);
+            base.Disable();
+        }
+
+        private void FrameworkOnUpdate(Framework framework) {
+            UpdateTargetStatus();
```
Comment (Caraxi) [inline Tweaks/UiAdjustment/LimitTargetStatusEffects.cs:44]: Can we get a try/catch on this

Comment (Aireil) [reply Tweaks/UiAdjustment/LimitTargetStatusEffects.cs]: Of couuurse!

**Pull request merged**